admin/data/migrations/migrations/20171019113308-edit-check-windows-status.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const config = require('../config')
+
+module.exports = {
+  up (db, next) {
+    mongoose.connect(config.mongodb.url, async error => {
+      if (error) {
+        next(new Error('Could not connect to mongodb: ' + error.message))
+      }
+      try {
+        await db.collection('checkwindows').update({}, {$set: {'deleted': false}}, {multi: true})
+        mongoose.disconnect(() => next())
+      } catch (error) {
+        console.log('ERROR', error)
+        next(error)
+      }
+    })
+  },
+
+  down (db, next) {
+    mongoose.connect(config.mongodb.url, async error => {
```

Comment:
Again, this mongoose.connect is not needed - you can just use the db var directly.  It's not wrong though.

Reply:
Please, let's keep it or refactor all the migrations. From now on if you don't want mongoose in the migrations scripts I won't use it but please don't make me rewrite this.